pool.go

```diff
@@ -258,6 +251,17 @@ func (p *pool) checkPool() {
 					complianceCidCallsTotalMetric.WithLabelValues("success").Add(1)
 				}
 
+				// see if it is to a main-tier node - if so find appropriate test node to test against.
+				// --- Mirroring
+				p.lk.RLock()
+				if p.th.NodeTier(msg.node) != tieredhashing.TierMain {
```

Comment:
should this be `p.th.NodeTier(node)` ?

Reply:
@willscott No, what we're checking here is that the original request that caused this mirroring was sent to a main node.

`node` is the node that we will send the mirrored traffic/compliance cid to.

Reply:
I thought `node` refers to the test node we are finding to test the main tier node against and `msg.node` is the main tier node.